l1-contracts/test/foundry/l1/unit/concrete/AddressAliasHelper/actualRefundRecipient.t.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {AddressAliasHelperSharedTest} from "./_AddressAliasHelper_Shared.t.sol";
+import {Test} from "forge-std/Test.sol";
+
+// Used to test when recipient is a contract
+contract TestContract {
+    // add this to be excluded from coverage report
+    function test() internal virtual {}
+}
+
+contract actualRefundRecipient is AddressAliasHelperSharedTest {
+    function test_When_recipientAddressIsNotZero() public {
+        address recipient = makeAddr("recipient");
+        address prevMessageSender = makeAddr("prevMessageSender");
+
+        address actualRecipient = addressAliasHelper.actualRefundRecipient(recipient, prevMessageSender);
+
+        assertEq(actualRecipient, recipient);
+    }
+
+    function test_When_recipientAddressIsZeroAndTxOriginIsPrevMsgSender() public {
+        address recipient = address(0);
+        address prevMessageSender = makeAddr("prevMessageSender");
+
+        vm.startBroadcast(prevMessageSender);
```

Comment:
startBroadcast should be followed by stopBroadcast (or if there are no other calls, you can just do `vm.broadcast()`

Reply:
Added stopBroadcast.